src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -160,7 +164,7 @@ def update_progress(cur_loss, iteration):
         'batch_size': batch_size,
         'training_loss': cur_loss,
         'training_time': training_time,
-        'max_iterations': max_iterations,
+        'num_iterations': num_iterations,
```

Comment:
I believe our other toolkits store `max_iterations` in the model state

Reply:
Done.